roles/teuthology/templates/teuthology.yaml

```diff
@@ -4,7 +4,7 @@ results_server: {{ paddles_address }}
 results_ui_server: {{ pulpito_address }}
 results_email: {{ teuthology_results_email|default('null') }}
 results_sending_email: {{ teuthology_results_sending_email|default('null') }}
-lab_domain: {{ lab_domain }}
+lab_domain: {{ lab_domain|default('teuthology') }}
```

Comment:
Could this go in https://github.com/ceph/ceph-cm-ansible/blob/master/roles/teuthology/defaults/main.yml instead please?

Reply:
Actually, it's fine.